core/api/views/cp_reports.py

```diff
@@ -278,7 +281,14 @@ def _archive_cp_report(self, instance):
     @transaction.atomic
     def put(self, request, *args, **kwargs):
         current_obj = self.get_object()
-        serializer = CPReportCreateSerializer(data=request.data)
+
+        # set event description
+        if not request.data.get("event_description"):
+            request.data["event_description"] = "Updated by user"
```

Comment:
Is the frontend expected to send an event description here? I think the backend has all the data here, so it makes sense to me for it to set the description, maybe after validating the data. Or are the users expected to fill this on the site?

Reply:
Miruna suggested that the frontend should send a text here, but I'm unsure about the specifics or whether the user should provide this text. 
To ensure the field isn't null and to provide some initial context, I added these default values. 
Once I merge this (since the backend is complete and ready for the frontend), I plan to ask the frontend team for more details on the helpdesk task.
What do you think?

Reply:
Sounds good.